azurerm/internal/services/monitor/resource_arm_monitor_metric_alert.go

```diff
@@ -397,7 +397,7 @@ func expandMonitorMetricAlertAction(input []interface{}) *[]insights.MetricAlert
 
 			actions = append(actions, insights.MetricAlertAction{
 				ActionGroupID:     utils.String(agID),
-				WebhookProperties: props,
+				WebHookProperties: props,
```

Comment:
I think we'll need to revert this?

Reply:
It does not compile otherwise.  
It seems to have changed as part of azure-sdk-for-gov39.3.0
in services/preview/monitor/mgmt/2019-06-01/insight/models.go:
```

type MetricAlertAction struct {
	// ActionGroupID - the id of the action group to use.
	ActionGroupID *string `json:"actionGroupId,omitempty"`
	// WebHookProperties - The properties of a webhook object.
	WebHookProperties map[string]*string `json:"webHookProperties"`
}

```
v0.39.3 is a required change.

But I can revert if you prefer?

Reply:
Nope i missed that your still updating the container service package, thought that was done in another PR my mistake 😅